test.js

```diff
@@ -43,4 +51,38 @@ describe('styled-jsx-plugin-sass', () => {
       `)
     )
   })
+
+  describe('node-sass options', () => {
+    let spy
+
+    before(() => {
+      spy = sinon.spy(sass, 'renderSync');
+      plugin = proxyquire('./', {
+        'node-sass': {
+          renderSync: spy
+        }
+      })
+    })
+
+    it('allows "precision" to be set', () => {
+      plugin('p { color: red; }', {
+        precision: 2
+      })
+      sinon.assert.calledWithMatch(spy, { precision: 2 })
```

Comment:
I don't think that it is necessary to assert that the function (`plugin`) that you called a few lines above is called with the arguments that you passed on purpose :) This means that you can remove `sinon` and `proxyquire`.

If anything I would use some fixtures and test that when called it produces the expected output.